include/zephyr/pm/device.h

```diff
@@ -66,22 +66,76 @@ enum pm_device_flag {
 
 /** @brief Device power states. */
 enum pm_device_state {
-	/** Device is in active or regular state. */
+	/**
+	 * @brief Device hardware is powered, and the device is needed by the system.
+	 *
+	 * @details The device should be enabled in this state. Any device driver API
+	 * may be called in this state.
+	 */
 	PM_DEVICE_STATE_ACTIVE,
 	/**
-	 * Device is suspended.
+	 * @brief Device hardware is powered, but the device is not needed by the
+	 * system.
 	 *
-	 * @note
-	 *     Device context may be lost.
+	 * @details The device should be put into its lowest internal power state,
```

Comment:
I still think the details about transitions here do not belong into the state definition, but fine, we will see how this will work out and can always change it.

Reply:
yes, we need to rethink this as a whole and consider this option.